examples/complete/main.tf

```diff
@@ -56,7 +94,26 @@ module "mongodb" {
   resource_group_id   = module.resource_group.resource_group_id
   mongodb_version     = var.mongodb_version
   instance_name       = "${var.prefix}-mongodb"
+  endpoints           = "private"
   region              = var.region
   key_protect_key_crn = module.key_protect_all_inclusive.keys["icd.${var.prefix}-mongodb"].crn
   tags                = var.resource_tags
+  cbr_rules = [
+    {
+      description      = "sample rule"
```

Comment:
In the UI this is the main identifier so update it to something like `${var.prefix}-mongodb access only from vpc`